lib/lightning/policies/workflows.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Lightning.Policies.Workflows do
+  @moduledoc """
+  The Bodyguard Policy module for Workflows API.
+  """
+  @behaviour Bodyguard.Policy
+
+  alias Lightning.Accounts.User
+  alias Lightning.Projects.Project
+  alias Lightning.Run
+
+  @type actions :: :access_write | :access_read
+  @spec authorize(actions(), User.t() | Run.t(), Project.t()) ::
+          :ok | {:error, :unauthorized}
+  def authorize(access, %User{} = user, project)
```

Comment:
I think this is a duplicate policy check. In liveview, we use the `ProjectUsers` module for this.

Reply:
to be continued here: https://github.com/OpenFn/lightning/issues/2822